airbyte-cdk/java/airbyte-cdk/core/src/test/kotlin/io/airbyte/cdk/integrations/destination/async/AsyncStreamConsumerTest.kt

```diff
@@ -43,79 +41,82 @@ import java.util.function.Consumer
 import java.util.stream.Collectors
 import java.util.stream.Stream
 import org.apache.commons.lang3.RandomStringUtils
-import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Assertions.assertTrue
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
 import org.mockito.ArgumentMatchers
 import org.mockito.Mockito
 
 class AsyncStreamConsumerTest {
-    private val RECORD_SIZE_20_BYTES = 20
-
-    private val SCHEMA_NAME = "public"
-    private val STREAM_NAME = "id_and_name"
-    private val STREAM_NAME2 = STREAM_NAME + 2
-    private val STREAM1_DESC: StreamDescriptor =
-        StreamDescriptor().withNamespace(SCHEMA_NAME).withName(STREAM_NAME)
-
-    private val CATALOG: ConfiguredAirbyteCatalog =
-        ConfiguredAirbyteCatalog()
-            .withStreams(
-                java.util.List.of(
-                    CatalogHelpers.createConfiguredAirbyteStream(
-                        STREAM_NAME,
-                        SCHEMA_NAME,
-                        Field.of("id", JsonSchemaType.NUMBER),
-                        Field.of("name", JsonSchemaType.STRING),
-                    ),
-                    CatalogHelpers.createConfiguredAirbyteStream(
-                        STREAM_NAME2,
-                        SCHEMA_NAME,
-                        Field.of("id", JsonSchemaType.NUMBER),
-                        Field.of("name", JsonSchemaType.STRING),
+    companion object {
+        private const val RECORD_SIZE_20_BYTES = 20
+        private const val SCHEMA_NAME = "public"
+        private const val STREAM_NAME = "id_and_name"
+        private const val STREAM_NAME2 = STREAM_NAME + 2
+        private val STREAM1_DESC: StreamDescriptor =
+            StreamDescriptor().withNamespace(SCHEMA_NAME).withName(STREAM_NAME)
+
+        private val CATALOG: ConfiguredAirbyteCatalog =
```

Comment:
Do we like non-const ALLCAPS names in Kotlin? Just curious, I don't care much.

Reply:
Kotlin wants you to only use capitals for constants.  These were originally static fields in the Java code, hence why they are capitalized.  We could/should convert these to lower camel to be consistent with Kotlin styling, but that can be done later if we agree with their style.

Reply:
Thanks for the explanation! I wasn't sure if it was just `const`s or not.